cluster/saltbase/salt/kube-master-addons/kube-master-addons.sh

```diff
@@ -22,23 +22,23 @@ while true; do
 
   if which docker 1>/dev/null 2>&1; then
 
-    timeout 30 docker load -i /srv/salt/kube-bins/kube-apiserver.tar 1>/dev/null 2>&1
+    timeout 120 docker load -i /srv/salt/kube-bins/kube-apiserver.tar 1>/dev/null 2>&1
```

Comment:
I increased the time for docker load to avoid unnecessary timeout when doing docker load. I have encounter an issue that some times 30 seconds is not enough for docker load